landavailability/api/management/commands/update_locations_nearest_bus.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand, CommandError
+from api.models import BusStop
+
+
+class Command(BaseCommand):
+    help = ''
+
+    def handle(self, *args, **options):
+        bss = BusStop.objects.all()
+        total_busses = len(bss)
```

Comment:
It's just a temporary script that I used to test the performance of the operation. I want to discuss with you where it would me more appropriate to call this method: should we call it as is at the end of a batch of bus imports? Or should we call it every time we .save() a Location?